daemon/config/builder.go

```diff
@@ -62,8 +63,8 @@ type BuilderGCConfig struct {
 
 // BuilderHistoryConfig contains history config for a buildkit builder
 type BuilderHistoryConfig struct {
-	MaxAge     int64 `json:",omitempty"`
-	MaxEntries int64 `json:",omitempty"`
+	MaxAge     bkconfig.Duration `json:",omitempty"`
```

Comment:
This change does not affect how the config is interpreted? (i.e. existing config still means the same?) because I _think_ this may be part of config options set through `daemon.json` 😅

Reply:
Existing options should work fine; you can see in https://github.com/moby/buildkit/commit/e06c96274ffe3bd777c202f121af151b36a9f57d that a custom deserializer is implemented to handle the existing int64 format, but also parse a Go `time.Duration` string.

Reply:
Ah, yes looks good. I didn't look yet, and was curious if the `* times.Second` was handled; I've (ahum) seen cases where that part was overlooked, and now the duration going from "seconds" to "nanoseconds" 😂